builds/misc/images-mqtt.yaml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+pr: none
+
+jobs:
+################################################################################
+  - job: linux_amd64
+################################################################################
+    displayName: Linux AMD64
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    variables:
+      imageName: azureiotedge/mqttd-linux-amd64
```

Comment:
Once the plan changed and you are pushing this image to edgebuilds, could you please make it in a format the rest of images follow?
The name can be more descriptive and contain tag that corresponds to a build
`azureiotedge/iotedge-mqtt-broker:{VERSION}-linux-{ARCH}`